defaults.lua

```diff
@@ -1,176 +1,154 @@
 -- To make configuration changes that persists between (nightly) releases,
--- copy defaults.lua to defaults.persistent.lua and make the changes there.
-
--- number of page turns between full screen refresh
--- default to do a full refresh on every 6 page turns
--- no longer needed
---DRCOUNTMAX = 6
+-- copy defaults.lua to defaults.custom.lua and make the changes there,
```

Comment:
Is it possible to provide a migration path for users with `defaults.persistent.lua`?

In a ideal world there should be a way to do the migration programatically to the new file and leave the old file there, unmodified except for a big comment on top saying `file no longer used, contents were migrated to defaults.custom.lua`.

Not sure if it worth the (your) time :p

Reply:
It's already handled in onetime_migration ;).

(Although I renamed the file after migration to make things clearer, to `defaults.legacy.lua`).